canonical-web-and-design/canonicalwebteam.store-api
# Add snapcraft tests

### Done
- Added tests for the Snapcraft API, using vcrpy-unittest library to save fixtures of API responses.
- Added GitHub Workflow badges.
- Added Codecov integration.
- Added GitHub Action to test once a day the live Snapcraft API

### Issue
https://github.com/canonical-web-and-design/snap-squad/issues/1203

### QA
Look at the GitHub check "Tests / test-python (pull_request)" in this PR and make sure it is working or pull the branch and do:
1. `poetry install`
2. `poetry run coverage run --source=. -m unittest discover tests`
